src/session_manager.rs

```diff
@@ -385,10 +384,9 @@ impl SessionManager {
         spawn(async move {
             send_log_level(client_id).await;
 
-            // TODO: Hopefully we'll support animated emoji at some point
-            let result = disable_animated_emoji(client_id, true).await;
+            let result = disable_animated_emoji(client_id, false).await;
```

Comment:
Is it really necessary to call this function given that we're not interested in disabling animated emoji any more?

Reply:
Animated emojis were disabled in older versions of telegrand, and that parameter is saved in cache